css-two-column-layout/styles.css

```diff
@@ -0,0 +1,102 @@
+.container {
+  align-items: center;
+  font-family: Karla, sans-serif;
+  padding: 10px;
+  justify-content: center;
+}
```

Comment:
`align-items` & `justify-content` are parent properties, which means they need to get applied where `display: flex` is. Since this doesn't have `display: flex`, these should be removed.